code/modules/mining/machine_redemption.dm

```diff
@@ -93,7 +93,15 @@
 		if(!M || !redemption_mat)
 			return 0
 
-		build_amount = min(build_amount, round(redemption_mat.amount / M))
+		var/smeltable_sheets = round(redemption_mat.amount / M)
+
+		if(!smeltable_sheets)
+			return 0
```

Comment:
Oh I see, I searched the whole file for the proc but missed it being used as value. The proc name is a bit misleading aswell, since a `can_*` could be read as a bool proc.